jcl/src/java.base/share/classes/java/lang/Class.java

```diff
@@ -1454,19 +1454,21 @@ private Method throwExceptionOrReturnNull(boolean throwException, String name, C
 Method getMethodHelper(
 	boolean throwException, boolean forDeclaredMethod, List<Method> methodList, String name, Class<?>... parameterTypes)
 	throws NoSuchMethodException {
-	Method result;
-	Method bestCandidate;
+	Method result = null;
+	Method resultFromSuperClass = null;
+	Method bestCandidate = null;
+	Method candidateFromSuperClass = null;
```

Comment:
There is no need to initialize java variable to `null` explicitly, `javac` has done that.

Reply:
Agreed and fixed.